Libplanet.Net/Swarm.BlockCandidate.cs

```diff
@@ -17,20 +17,20 @@ public partial class Swarm<T>
             var checkInterval = TimeSpan.FromMilliseconds(10);
             while (!cancellationToken.IsCancellationRequested)
             {
-                if (BlockCandidateTable.Any())
+                if (BlockCandidateTable.Count > 0)
                 {
                     BlockHeader tipHeader = BlockChain.Tip.Header;
-                    SortedList<long, Block<T>> blocks =
```

Comment:
I want to make sure this list is indeed sorted. Have we a way to prohibit adding a non-sorted list in this code? Without `SortedList<K, V>`?

Reply:
Not really. However, a list being sorted is not a sufficient condition (for consumption), nor is it reflective of the actual constraints (for `BlockCandidateTable`), as I have already mentioned in the PR message above. 😶

Reply:
At first, I'm not sure about just guarding in `BlockCandidateTable.Add()` and leaving the inner data structure to `List<T>`. Because if left to `List<T>`, other contributors can mistake. (e.g., Make new non-sorted append I guess.)
I'm curious about your opinion about this.

Reply:
I agree, hence the need for a new `class`. If there isn't any logic issues, once this gets merged I'll work on it ASAP. 😗